libs/blocks/chart/chart.js

```diff
@@ -298,10 +331,12 @@ export const getChartOptions = (chartType, data, colors, size, chart) => {
     },
     tooltip: {
       show: true,
-      formatter: isBar
-        ? (params) => barTooltipFormatter(params, units[0])
-        : (params) => tooltipFormatter(params, units),
-      trigger: isBar ? 'item' : 'axis',
+      formatter: ((params) => {
+        if (isBar) return barTooltipFormatter(params, units[0]);
+        if (chartType === 'pie') return pieTooltipFormatter(params, units[0]);
```

Comment:
I believe we have `isBar` and `isColumn` variables defined above. Any particular reason why you decided not to go with another variable to store the conditional expression this time since we're seeing it also used in a couple of places?